### PR TITLE
Document TryParse Span method overloads for System.B-G

### PR DESCRIPTION
**Pull request by carlossanlop**
Types: Boolean, Byte, DateTime, DateTimeoffset, Decimal, Double, Enum, Guid

### CODE REVIEW

File: xml/System/Boolean.xml [hunk 1]
```diff
@@ -1822,10 +1822,10 @@
...
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">A span containing the characters representing the value to convert.</param>
+        <param name="result">When this method returns, if the conversion succeeded, contains <see langword="true" /> if <paramref name="value" /> is equal to <see cref="F:System.Boolean.TrueString" /> or <see langword="false" /> if <paramref name="value" /> is equal to <see cref="F:System.Boolean.FalseString" />. If the conversion failed, contains <see langword="false" />. The conversion fails if <paramref name="value" /> is <see langword="null" /> or is not equal to the value of either the <see cref="F:System.Boolean.TrueString" /> or <see cref="F:System.Boolean.FalseString" /> field.</param>
+        <summary>Tries to convert the specified readonly char span representation of a logical value to its <see cref="T:System.Boolean" /> equivalent.</summary>
```
Comment (carlossanlop) [inline xml/System/Boolean.xml]:
```suggestion
        <summary>Tries to convert the specified span representation of a logical value to its <see cref="T:System.Boolean" /> equivalent.</summary>
```

File: xml/System/Byte.xml [hunk 2]
```diff
@@ -2351,10 +2351,10 @@
         <Parameter Name="result" Type="System.Byte" RefType="out" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">A span containing the characters representing the number to convert.</param>
+        <param name="result">When this method returns, contains the <see cref="T:System.Byte" /> value equivalent to the number contained in <paramref name="s" /> if the conversion succeeded, or zero if the conversion failed. This parameter is passed uninitialized; any value originally supplied in <paramref name="result" /> will be overwritten.</param>
+        <summary>Tries to convert the readonly char span representation of a number to its <see cref="T:System.Byte" /> equivalent, and returns a value that indicates whether the conversion succeeded.</summary>
```
Comment (carlossanlop) [inline xml/System/Byte.xml]:
```suggestion
        <summary>Tries to convert the span representation of a number to its <see cref="T:System.Byte" /> equivalent, and returns a value that indicates whether the conversion succeeded.</summary>
```

File: xml/System/Byte.xml [hunk 3]
```diff
@@ -2471,12 +2470,12 @@
...
+        <param name="provider">An object that supplies culture-specific formatting information about <paramref name="s" />. If <paramref name="provider" /> is <see langword="null" />, the thread current culture is used.</param>
+        <param name="result">When this method returns, contains the 8-bit unsigned integer value equivalent to the number contained in <paramref name="s" /> if the conversion succeeded, or zero if the conversion failed. The conversion fails if the <paramref name="s" /> parameter is <see langword="null" /> or <see cref="F:System.String.Empty" />, is not of the correct format, or represents a number less than <see cref="F:System.Byte.MinValue" /> or greater than <see cref="F:System.Byte.MaxValue" />. This parameter is passed uninitialized; any value originally supplied in <paramref name="result" /> will be overwritten.</param>
+        <summary>Converts the readonly char span representation of a number in a specified style and culture-specific format to its <see cref="T:System.Byte" /> equivalent. A return value indicates whether the conversion succeeded or failed.</summary>
```
Comment (carlossanlop) [inline xml/System/Byte.xml]:
```suggestion
        <summary>Converts the span representation of a number in a specified style and culture-specific format to its <see cref="T:System.Byte" /> equivalent. A return value indicates whether the conversion succeeded or failed.</summary>
```

File: xml/System/DateTime.xml [hunk 4]
```diff
@@ -8222,12 +8221,12 @@ The value of the current <xref:System.DateTime> object is formatted using the pa
...
+        <param name="styles">A bitwise combination of enumeration values that defines how to interpret the parsed date in relation to the current time zone or the current date. A typical value to specify is <see cref="F:System.Globalization.DateTimeStyles.None" />.</param>
+        <param name="result">When this method returns, contains the <see cref="T:System.DateTime" /> value equivalent to the date and time contained in <paramref name="s" />, if the conversion succeeded, or <see cref="F:System.DateTime.MinValue" /> if the conversion failed. The conversion fails if the <paramref name="s" /> parameter is <see langword="null" />, is an empty string (""), or does not contain a valid string representation of a date and time. This parameter is passed uninitialized.</param>
+        <summary>Converts the specified readonly char span representation of a date and time to its <see cref="T:System.DateTime" /> equivalent using the specified culture-specific format information and formatting style, and returns a value that indicates whether the conversion succeeded.</summary>
```
Comment (carlossanlop) [inline xml/System/DateTime.xml]:
```suggestion
        <summary>Converts the span representation of a date and time to its <see cref="T:System.DateTime" /> equivalent using the specified culture-specific format information and formatting style, and returns a value that indicates whether the conversion succeeded.</summary>
```

File: xml/System/DateTimeOffset.xml [hunk 5]
```diff
@@ -5405,10 +5405,10 @@ Me.ToUtcDateTime().ToFileTime()
...
       </Parameters>
       <Docs>
-        <param name="input">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="input">A span containing the characters representing the date and time to convert.</param>
+        <param name="result">When the method returns, contains the <see cref="T:System.DateTimeOffset" /> equivalent to the date and time of <paramref name="input" />, if the conversion succeeded, or <see cref="F:System.DateTimeOffset.MinValue" />, if the conversion failed. The conversion fails if the <paramref name="input" /> parameter is <see langword="null" /> or does not contain a valid string representation of a date and time. This parameter is passed uninitialized.</param>
+        <summary>Tries to convert a specified readonly char span representation of a date and time to its <see cref="T:System.DateTimeOffset" /> equivalent, and returns a value that indicates whether the conversion succeeded.</summary>
```
Comment (carlossanlop) [inline xml/System/DateTimeOffset.xml]:
```suggestion
        <summary>Tries to convert a specified span representation of a date and time to its <see cref="T:System.DateTimeOffset" /> equivalent, and returns a value that indicates whether the conversion succeeded.</summary>
```

File: xml/System/DateTimeOffset.xml [hunk 6]
```diff
@@ -5518,12 +5517,12 @@ Me.ToUtcDateTime().ToFileTime()
...
-        <returns>To be added.</returns>
+        <param name="input">A span containing the characters representing the date and time to convert.</param>
+        <param name="formatProvider">An object that provides culture-specific formatting information about <paramref name="input" />.</param>
+        <param name="styles">A bitwise combination of enumeration values that indicates the permitted format of <paramref name="input" />.</param>
+        <param name="result">When the method returns, contains the <see cref="T:System.DateTimeOffset" /> value equivalent to the date and time of <paramref name="input" />, if the conversion succeeded, or <see cref="F:System.DateTimeOffset.MinValue" />, if the conversion failed. The conversion fails if the <paramref name="input" /> parameter is <see langword="null" /> or does not contain a valid string representation of a date and time. This parameter is passed uninitialized.</param>
+        <summary>Tries to convert a specified string representation of a date and time to its <see cref="T:System.DateTimeOffset" /> equivalent, and returns a value that indicates whether the conversion succeeded.</summary>
```
Comment (carlossanlop) [inline xml/System/DateTimeOffset.xml]:
```suggestion
        <summary>Tries to convert a specified span representation of a date and time to its <see cref="T:System.DateTimeOffset" /> equivalent, and returns a value that indicates whether the conversion succeeded.</summary>
```

File: xml/System/Decimal.xml [hunk 7]
```diff
@@ -7232,10 +7232,10 @@
...
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">A span containing the characters representing the number to convert.</param>
+        <param name="result">When this method returns, contains the <see cref="T:System.Decimal" /> number that is equivalent to the numeric value contained in <paramref name="s" />, if the conversion succeeded, or zero if the conversion failed. The conversion fails if the <paramref name="s" /> parameter is <see langword="null" /> or <see cref="F:System.String.Empty" />, is not a number in a format compliant with <paramref name="style" />, or represents a number less than <see cref="F:System.Decimal.MinValue" /> or greater than <see cref="F:System.Decimal.MaxValue" />. This parameter is passed uininitialized; any value originally supplied in <paramref name="result" /> is overwritten.</param>
+        <summary>Converts the readonly char span representation of a number to its <see cref="T:System.Decimal" /> equivalent using the specified style and culture-specific format. A return value indicates whether the conversion succeeded or failed.</summary>
```
Comment (carlossanlop) [inline xml/System/Decimal.xml]:
```suggestion
        <summary>Converts the span representation of a number to its <see cref="T:System.Decimal" /> equivalent using the specified style and culture-specific format. A return value indicates whether the conversion succeeded or failed.</summary>
```

File: xml/System/Decimal.xml [hunk 8]
```diff
@@ -7360,12 +7360,12 @@
...
+        <param name="provider">An object that supplies culture-specific parsing information about <paramref name="s" />.</param>
+        <param name="result">When this method returns, contains the <see cref="T:System.Decimal" /> number that is equivalent to the numeric value contained in <paramref name="s" />, if the conversion succeeded, or zero if the conversion failed. The conversion fails if the <paramref name="s" /> parameter is <see langword="null" /> or <see cref="F:System.String.Empty" />, is not a number in a format compliant with <paramref name="style" />, or represents a number less than <see cref="F:System.Decimal.MinValue" /> or greater than <see cref="F:System.Decimal.MaxValue" />. This parameter is passed uininitialized; any value originally supplied in <paramref name="result" /> is overwritten.</param>
+        <summary>Converts the readonly char span representation of a number to its <see cref="T:System.Decimal" /> equivalent using the specified style and culture-specific format. A return value indicates whether the conversion succeeded or failed.</summary>
```
Comment (carlossanlop) [inline xml/System/Decimal.xml]:
```suggestion
        <summary>Converts the span representation of a number to its <see cref="T:System.Decimal" /> equivalent using the specified style and culture-specific format. A return value indicates whether the conversion succeeded or failed.</summary>
```

File: xml/System/Double.xml [hunk 9]
```diff
@@ -3645,10 +3645,10 @@
...
+        <param name="s">A span containing the characters representing the number to convert.</param>
+        <param name="result">When this method returns, contains a double-precision floating-point number equivalent of the numeric value or symbol contained in <paramref name="s" />, if the conversion succeeded, or zero if the conversion failed. The conversion fails if the <paramref name="s" /> parameter is <see langword="null" /> or <see cref="F:System.String.Empty" />, is not in a format compliant with <paramref name="style" />, represents a number less than <see cref="F:System.SByte.MinValue" /> or greater than <see cref="F:System.SByte.MaxValue" />, or if <paramref name="style" /> is not a valid combination of <see cref="T:System.Globalization.NumberStyles" /> enumerated constants. This parameter is passed uninitialized; any value originally supplied in <paramref name="result" /> will be overwritten.</param>
+        <summary>Converts the readonly char span representation of a number in a specified style and culture-specific format to its double-precision floating-point number equivalent. A return value indicates whether the conversion succeeded or failed.</summary>
```
Comment (carlossanlop) [inline xml/System/Double.xml]:
```suggestion
        <summary>Converts the span representation of a number in a specified style and culture-specific format to its double-precision floating-point number equivalent. A return value indicates whether the conversion succeeded or failed.</summary>
```

File: xml/System/Double.xml [hunk 10]
```diff
@@ -3780,12 +3780,12 @@
...
```
Comment (carlossanlop) [inline xml/System/Double.xml]:
```suggestion
        <summary>Converts the span representation of a number in a specified style and culture-specific format to its double-precision floating-point number equivalent. A return value indicates whether the conversion succeeded or failed.</summary>
```

**Comment by carlossanlop** [comment]
Thank you for the approval @Thraka. The build passed without any warnings. Would you mind merging it? Or as @BillWagner would say, should we :shipit: ?

**Comment by gewarren** [comment]
> Thank you for the approval @Thraka. The build passed without any warnings. Would you mind merging it? Or as @BillWagner would say, should we :shipit: ?

Someone needs to make Bill a ship-it squirrel t-shirt.

**Comment by carlossanlop** [comment]
> Someone needs to make Bill a ship-it squirrel t-shirt.

Or his GitHub picture 😄

**Pull request merged**